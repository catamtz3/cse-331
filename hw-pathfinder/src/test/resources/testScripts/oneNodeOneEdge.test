CreateGraph oneNode

AddNode oneNode a

AddEdge oneNode a a 1.0

ListNodes oneNode

FindPath oneNode a b