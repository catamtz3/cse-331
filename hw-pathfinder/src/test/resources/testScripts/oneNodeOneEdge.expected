created graph oneNode

added node a to oneNode

added edge 1.000 from a to a in oneNode

oneNode contains: [a]

unknown: b